.babelrc

```diff
@@ -1,6 +1,11 @@
 {
   "presets": [
-    "@babel/preset-typescript",
+    [
+      "@babel/preset-typescript",
+      {
+        "shippedProposals": true
+      }
+    ],
```

Comment:
Comment for me, might need to remove this, I think I broke something in webpack and added this trying to fix it